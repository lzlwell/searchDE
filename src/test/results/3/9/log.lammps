LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26699 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1090.3482   -1090.3482      17.1317           10           50 
       1    -1116.763    -1116.763      17.1317           10           50 
       2   -1137.3817   -1137.3817      17.1317           10           50 
       3   -1150.1824   -1150.1824      17.1317           10           50 
       4   -1158.2791   -1158.2791      17.1317           10           50 
       5   -1162.5347   -1162.5347      17.1317           10           50 
       6    -1166.166    -1166.166      17.1317           10           50 
       7   -1167.1046   -1167.1046      17.1317           10           50 
       8    -1167.653    -1167.653      17.1317           10           50 
Loop time of 0.332702 on 1 procs for 8 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1090.34820174     -1167.10457702     -1167.65296969
  Force two-norm initial, final = 178.875 26.465
  Force max component initial, final = 105.947 18.7211
  Final line search alpha, max atom move = 0.00214296 0.0401185
  Iterations, force evaluations = 8 12

Pair  time (%) = 0.158999 (47.7903)
Neigh time (%) = 0.170408 (51.2195)
Comm  time (%) = 0.00132728 (0.398938)
Outpt time (%) = 0.000109911 (0.0330358)
Other time (%) = 0.00185752 (0.558313)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2339 ave 2339 max 2339 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46704 ave 46704 max 46704 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46704
Ave neighs/atom = 155.68
Neighbor list builds = 8
Dangerous builds = 7
min_modify line quadratic

clear
