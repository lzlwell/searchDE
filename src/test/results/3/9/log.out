LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26699 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1090.3482   -1090.3482      17.1317           10           50 
       1    -1116.763    -1116.763      17.1317           10           50 
       2   -1137.3817   -1137.3817      17.1317           10           50 
       3   -1150.1824   -1150.1824      17.1317           10           50 
       4   -1158.2791   -1158.2791      17.1317           10           50 
       5   -1162.5347   -1162.5347      17.1317           10           50 
       6    -1166.166    -1166.166      17.1317           10           50 
       7   -1167.1046   -1167.1046      17.1317           10           50 
       8    -1167.653    -1167.653      17.1317           10           50 
Loop time of 0.332702 on 1 procs for 8 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1090.34820174     -1167.10457702     -1167.65296969
  Force two-norm initial, final = 178.875 26.465
  Force max component initial, final = 105.947 18.7211
  Final line search alpha, max atom move = 0.00214296 0.0401185
  Iterations, force evaluations = 8 12

Pair  time (%) = 0.158999 (47.7903)
Neigh time (%) = 0.170408 (51.2195)
Comm  time (%) = 0.00132728 (0.398938)
Outpt time (%) = 0.000109911 (0.0330358)
Other time (%) = 0.00185752 (0.558313)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2339 ave 2339 max 2339 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46704 ave 46704 max 46704 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46704
Ave neighs/atom = 155.68
Neighbor list builds = 8
Dangerous builds = 7
